specs/logging_change.adoc

```diff
@@ -0,0 +1,147 @@
+// vim: tw=avored-markdown/preview.ht79
+
+# Changes in tendrl modules after introduction of new logging framework
+
+Since a new logging module is being introduced (https://github.com/Tendrl/specifications/pull/94), after its implementation
+there will be requirement of changes in all the modules of Tendrl where logging is used.
+This file proposes all those changes and how they will be made.
+
+
+## Problem description
+
+Many modules in Tendrl have a functionality of logging different types of situations/use cases such as logging (info, errors, warning etc.).
+Till now these were implemented by importing the Python standard library's logging module then calling logger functions to handle the logging.
+After implementation of new logging module there will be requirement of changes in all the modules of Tendrl where logging is used.
+
+#### Use Cases
+
+Logs and operation logs need be treated as messages.
+
+#### Proposed change
+
+* *Importing logging and calling logger functions* will be replaced by:
+
+** Importing Message and Event class from commons
+** Creating a Message object by passing the required arguments to it.
+** After the Message object is created, using it as an argument to create an Event object which will write the message to socket
+* There will be 3 arguments that will be mandatory to create the Message object:
+ ** priority  <String>
+ ** publisher <String>
+ *** eg. CEPH-INTEGRATION, GLUSTER-INTEGRATION, NODE-AGENT, COMMONS, ALERTING, PERFORMANCE-MONITORING
+ ** payload   <Json>
+ *** If the message is of a log type the payload should look like :
+
+ {
+    "message" : "User Message"
```

Comment:
Don't mention type, just specify  like message format  for normal log, message format for operation udtates